Out-In-The-Open-June-2022 (#359)
okfde/codefor.de: content/blog/2022-07-07-out-in-the-open-june-2022.md

---
title: "Alles Fassade? – Out in the Open Juni 2022"

> author: "Klara Juhl - OKLab Osnabrück, Jörg Reichert - OKLab Leipzig, et al."

Reviewer: Wie ist denn die Policy bei der Schreibweise? Ich hätte `OK Lab Osnabrück` (mit Leerzeichen) gedacht, das ist bei den jüngsten Blogposts aber schon unterschiedlich.

date: '2022-07-07T12:00:00+02:00'
excerpt: Monatlich erscheinender Überblick über Entwicklungen in der Open Data und Civic Tech Szene
topic: community
images:
- imgname: out-in-the-open-2022-06-header.jpg
  attribution: Foto von [Linnaea Mallette](https://publicdomainpictures.net/en/browse-author.php?a=18382) auf [publicdomainpictures.net](https://publicdomainpictures.net/en/view-image.php?image=265257&picture=neon-open-sign)
og_image: out-in-the-open-2022-06-header.jpg
type: blog
---

## Verschwörhaus

> Der im [letzten Out-In-The-Open-Blogbeitrag](https://codefor.de/blog/out-in-the-open-may-2022/) bereits erwähnte Konflikt um das Verschwörhaus Ulm ist weiter eskaliert: Nachdem die Ehrenamtlichen sich nicht darauf einlassen wollten, die jahrelang aufgebaute Marke "Verschwörhaus" (und den mit ihr verbundenen Werte-Versprechen) der Stadt zu überlassen, wurden sie nun aus den bisherigen Räumlichkeiten [geworfen](https://verschwoerhaus.de/das-verschwoerhaus-zieht-um/).

Reviewer 2: https://gohugo.io/content-management/shortcodes/#ref-and-relref

Sorgt dafür, dass Hugo immer den richtigen Link hinschreibt

```suggestion
Der im [letzten Out-In-The-Open-Blogbeitrag]({{< ref "/content/blog/2022-06-09-out-in-the-open-may-2022.md" >}}) bereits erwähnte Konflikt um das Verschwörhaus Ulm ist weiter eskaliert: Nachdem die Ehrenamtlichen sich nicht darauf einlassen wollten, die jahrelang aufgebaute Marke "Verschwörhaus" (und den mit ihr verbundenen Werte-Versprechen) der Stadt zu überlassen, wurden sie nun aus den bisherigen Räumlichkeiten [geworfen](https://verschwoerhaus.de/das-verschwoerhaus-zieht-um/).
```

## Transparenzgesetz(e)

Ein Zusammenschluss mehrerer Initiativen und Vereine stellt einen Entwurf für das Bundestransparenzgesetz vor - darunter z. B. [FragDenStaat](https://fragdenstaat.de/) und die [Deutsche Gesellschaft für Informationsfreiheit](https://dgif.de/). Der Entwurf kann nicht nur [öffentlich eingesehen, sondern auch kommentiert werden](https://consul.mehr-demokratie.info/transparenzgesetz). Jede:r kann die eigene Sichtweise einbringen - egal ob "Volljurist:in oder politisch mehr oder weniger interessiert".

Die Londoner Verkehrsgesellschaft LNER wird auf dem Weg zu mehr Transparenz kreativ: [Das Unternehmen stellte zunächst eine IFG-Anfrage an sich selbst](https://twitter.com/fragdenstaat/status/1536759444922675201), um diese prompt zu beantworten und Dokumente an zentraler Stelle zu veröffentlichen.

> Auch Sachsen soll nun endlich ein Transparenzgesetz bekommen, aber wie FragDenStaat [bemängelt](https://twitter.com/fragdenstaat/status/1542431477467648000), liegt die die Kostendeckelung mit 2500€ fünfmal so hoch wie im Bund. Die Kostenuntergrenze von 600 Euro (d.h. erst ab dieser Höhe dürfen die Kosten dem Antragsteller in Rechnung gestellt werden) ist allerdings eine bessere Regelung als im für den Bund geltenden Transparenzgesetz.  

Reviewer: ```suggestion
Auch Sachsen soll nun endlich ein Transparenzgesetz bekommen, aber wie FragDenStaat [bemängelt](https://twitter.com/fragdenstaat/status/1542431477467648000), liegt die Kostendeckelung mit 2500€ fünfmal so hoch wie im Bund. Die Kostenuntergrenze von 600 Euro (d.h. erst ab dieser Höhe dürfen die Kosten dem Antragsteller in Rechnung gestellt werden) ist allerdings eine bessere Regelung als im für den Bund geltenden Transparenzgesetz.  
```

## APIs
Bei Dashboards, wie z.B. dem der [Berliner Morgenpost](https://www.morgenpost.de/infografik/#/grafik/62b5d08a0b908d73fcea4692), das den Strombedarf der Berliner aufgeteilt nach Gebäudeblöcken zeigt, [fragt sich datenschatz, woher die Daten stammen](https://twitter.com/datenschatz/status/1537411609982816257) und [listet](https://twitter.com/datenschatz/status/1537412468351328256) einige, der meist nicht offiziell dokumentierten APIs auf.  

Besser mancht es die API des Bundestages, deren Dokumentation ist nun auch in englischer Sprache verfügbar [[PDF](https://data4parliaments.org/wp-content/uploads/2022/06/API-Documentation-for-DIP_EN.pdf)]. 

Dennoch wird der Nutzen öffentlicher APIs in vielen anderen Ländern deutlich besser gehoben als hierzulande, geht aus diesem [Report](https://de.public.io/report-post/offentliche-apis-und-govtech-mit-interoperabilitat-innovation-fordern) hervor. Marco Wiedemann [fordert](https://twitter.com/mariosorg/status/1541766902174949381) daher, dass [govdata](https://www.govdata.de/) personell aufgestockt werden sollte, um auch APIs zu listen und besser auffindbar zu machen.

> Einen Anfang macht schon mal [Thomas Tursic](https://twitter.com/tursics), der gerade eben zu govdata [gewechselt](https://twitter.com/Hamburg67/status/1542784443097702400) ist.

Reviewer 2: ```suggestion
Einen Anfang macht schon mal [Thomas Tursics](https://twitter.com/tursics), der gerade eben zu govdata [gewechselt](https://twitter.com/Hamburg67/status/1542784443097702400) ist.
```
